src/app/components/dashboard/flagwidget/TableEntryFragments.jsx

```diff
@@ -41,7 +41,7 @@ const TableEntry = compose(
       selectedLang === f.first(Langtags)
         ? doto(
             table,
-            f.get("translationStatus"),
+            f.get(["translationStatus"]),
```

Comment:
Bitte das Array entfernen. Hast du mal den Linter drüberlaufen lassen wg. Indents?

Reply:
Linter sagt das alles in ordnung ist ^^